corehq/ex-submodules/couchforms/tests/test_analytics.py

```diff
@@ -135,7 +135,7 @@ def setUpClass(cls):
         @patch('corehq.apps.es.es_query.ES_META', TEST_ES_META)
         @patch('corehq.elastic.ES_META', TEST_ES_META)
         def create_form_and_sync_to_es(received_on):
-            with process_pillow_changes('XFormToElasticsearchPillow'):
+            with process_pillow_changes('kafka-xform-ucr-es', {'skip_ucr': True}):
```

Comment:
I notice you changed the function names but the ID's are still the previous naming scheme. Would be good to keep them consistent.

Reply:
Yeah, I tried to follow old scheme. I can update, I just need to do some custom thing to update staging ones since they are already live with new names.

Reply:
Hey, I have addressed renaming of pillows and checkpoint IDs per above suggestions. Please see if they are alright. 
(It's a pretty involved process to change checkpoint ids since it's deployed on staging already. So, I will do it once no further feedback on naming.)